src/models/typeDefinition.ts

```diff
@@ -193,7 +193,14 @@ export class TypeDefinitionObjectProperty extends TypeDefinitionProperty {
         }
 
         if (contract.type === "array") {
```

Comment:
Regarding this block:
```
        if (contract.type === "array") {
            if (contract.items.$ref) {
```

I think we should add check for presence `contract.items` as well, to minimize chance of breaking the page.

Reply:
Will this check work? Because items are required for type array (accordingly to OpenAPI schema definition documentation).